Generated data was too big for Github. Generate then by running main.py
